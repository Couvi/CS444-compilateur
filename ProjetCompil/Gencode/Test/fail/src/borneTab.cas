program
    T : array[1..5] of REAL ; -- Ok
begin
    T[0] := 0 ;
end.