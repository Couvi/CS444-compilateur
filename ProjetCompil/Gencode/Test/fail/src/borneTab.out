ERREUR Interval Out Of Bound
